File: app_java/src/main/java/vlab/android/architecture/model/UserInfo.java
package vlab.android.architecture.model;

import vlab.android.architecture.repository.source.remote.response.UserResponse;

/**
 * Created by Vinh Tran on 2/15/18.
 */

public class UserInfo {

    String userName;

    public UserInfo() {
    }

    public UserInfo(UserResponse userResponse) {
        this.userName = userResponse.getName();
    }

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }
}

File: app_kotlin/src/main/java/vlab/android/architecture/model/UserInfo.kt
package vlab.android.architecture.model

import vlab.android.architecture.repository.source.remote.response.UserResponse

/**
 * Created by Vinh Tran on 2/15/18.
 */

class UserInfo {

    var userName: String?

    constructor(userResponse: UserResponse) {
        this.userName = userResponse.name
    }
}
